### ISSUE TITLE
Embed Coinlist video 

### ISSUE THREAD

**Issue by pennyandrews** [vegaprotocol/vega.xyz#396]
## The Chore

We want to use the video we made for Coinlist on the website as it does such a great job of explaining Vega.  

## Tasks

- [ ] Decide where it should be placed on the site (assuming landing page but where about in the context of existing content)
- [ ] Work out the best technical solution that doesn't involve third party cookies or tracking (and agree approach with Barney)
- [ ] Implement

**Comment by pennyandrews** [comment]
A few notes having looked at your wee POC @goslingo 

- Could we choose (or make) a better thumbnail for what you see before you play the video
- Would it be possible to cut the end of the video so we don't see the CTA?
- Can we merge (or replicate) the PR for this with the one to change the descriptor so that we can see it all together?

**Comment by pennyandrews** [comment]
Blocked by #407 as want to release these improvements at the same time

**Comment by pennyandrews** [comment]
Superseded by landing page redesign
